src/rabbitmq.js

```diff
@@ -271,15 +271,14 @@ class RabbitMQ {
     return Promise.try(() => {
       const queueName = `${this.name}.${queue}`
       const bufferContent = this._validatePublish(queue, content, 'tasks')
-      const jobOpts = {
-        headers: {
-          publisher: this.name
-        }
+      const jobMeta = {
+        appId: this.name,
+        timestamp: Date.now()
```

Comment:
Also let's call this published timestamp so we can split it from received timestamp